### DISCUSSION TITLE
After modal closed, table disappear.

### DISCUSSION THREAD

**Discussion by soap** [rappasoft/laravel-livewire-tables#1311] [Q&A]
I have a livewire component responsible for add new record using Bootstrap Modal. I use Laravel Livewire table for display list of records. When I saved the record and close a modal using event, the table disappear from the display. This may be a simple solution that I do not know.

Any suggestion?

**Comment by Loydtafireyi** [comment]
Code examples might be useful

**Comment by lrljoe** [comment]
It's likely that you've got an unclosed element somewhere, but as @Loydtafireyi  says, a gist would be very useful!

**Comment by lrljoe** [comment]
@soap , please test with the v3-develop branch, think I may have fixed the bug which is with how LW3 deals with nested components

**Reply by soap** [reply]
OK I will try it. Sorry for late response.

**Comment by lrljoe** [comment]
This should be fixed in v3-develop.

Anyone experiencing this issue at a later date, please open a new discussion rather than replying to this one.
